locale/en/tag_names.json

```diff
@@ -0,0 +1,39 @@
+{
+    "@metadata": {
+		"authors": [
+			"Susana Cárdenas Molinar",
+			"Jason Sherman",
+			"Sam Walton"
+		],
+		"locale": "en",
+		"message-documentation": "qqq"
+	},
+    "language_translation": "language",
+    "multidisciplinary_translation": "multidisciplinary",
+    "physical-science_translation": "physical-science",
```

Comment:
I think these tags had no spaces/capital letters in the past for technical reasons. I think they could all be properly capitalised and have hyphens removed in favour of spaces (so in this case **Physical science**).

Reply:
Done!